libs/designsystem/link/src/link.component.stories.ts

```diff
@@ -0,0 +1,30 @@
+import type { Meta, StoryObj } from '@storybook/angular';
+import { moduleMetadata } from '@storybook/angular';
+
+import { LinkComponent } from '@kirbydesign/designsystem/link';
+
+import { LinkExampleButtonComponent } from '~/app/examples/link-example/examples/button';
+
+const meta: Meta<LinkComponent> = {
+  component: LinkComponent,
+  title: 'Components / Link',
+  decorators: [
+    moduleMetadata({
+      imports: [LinkExampleButtonComponent],
+    }),
+  ],
+};
+export default meta;
+type Story = StoryObj<LinkComponent>;
+
+export const Link: Story = {
+  render: () => ({
+    template: `<button kirby-link>Link-style button</button>`,
+  }),
+};
+
+export const CookbookExample: Story = {
+  render: () => ({
+    template: `<cookbook-link-example-button></cookbook-link-example-button>`,
+  }),
+};
```

Comment:
100% get why you added this, but generally we limit cookbook-examples in storybook to those that contain multiple examples with better bang for the buck, like the header example page: https://cookbook.kirby.design/#/examples/header